pytorch_lightning/callbacks/progress/rich_progress.py

```diff
@@ -190,6 +190,23 @@ class RichProgressBarTheme:
     metrics: Union[str, Style] = "white"
 
 
+def _detect_light_colab_theme() -> bool:
```

Comment:
should it use the way we have in tqdm progress bar to check for colab environment?

Reply:
We could. But by using this approach, we could scale this utility for updating themes for jupyter notebooks, kaggle, running on terminal in future, if required.